Update api documentation to add labels in commit

vdemeester opened the pull request:
It is already possible to set labels at commit when using the API, but it is not present in the API documentation. This is a little bit related to #13178 but this just documentation of a current working behavior.

This PR adds the _Labels_ part in the "Example Request" in the documentation and an integration test too, to validate that this works.

Signed-off-by: Vincent Demeester vincent@sbr.pm

jfrazelle commented:
LGTM

thaJeztah commented:
Can you add this to the 1.20 API as well? That file was just added

calavera commented:
LGTM, moving to docs review.

thaJeztah commented:
@calavera oops, my bad, thought it was in docs review already

@vdemeester if you can address my earlier comment, then LGTM

vdemeester commented:
@thaJeztah rebased to the current master to have the 1.20 api file and updated it :wink:.

vdemeester committed:
Update api documentation to add labels in commit

It is already possible to set labels at commit when using the API. But
it is not present in the API documentation. Added an integration test
too, to validate this work (and will be in the future).

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

vdemeester commented:
Hum, experimental doesn't like me :-)

thaJeztah commented:
I gave the experimental build a kick.

Thanks for updating, LGTM

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit for a quick review

jamtur01 commented:
LGTM

thaJeztah commented:
All green, merging.

Thanks @vdemeester!
